distmv/normal.go

```diff
@@ -21,7 +21,8 @@ type Normal struct {
 	mu    []float64
 	sigma *mat64.SymDense
```

Comment:
sigma can be deleted since it's not used anywhere and make chol and (lower) concrete rather than pointer types and lose the &{} lines in NewNormal.